### DISCUSSION TITLE
xray最快的代理方式？

### DISCUSSION THREAD

**Discussion by smallprogram** [XTLS/Xray-core#709] [Q&A]
我目前在用的是 xray前置(Vless+TCP+XTLS+)+NginxWeb的方式，感觉很不错。我非常好奇，其余的这些代理方式，那种是速度最快的。可有大佬做过评测？
||
|-|
|Shadowsocks-AEAD|
|Add Shadowsocks-AEAD|
|Shadowsocks-TCP|
|Socks5-TLS|
|Trojan-TCP-TLS (minimal)|
|VLESS-GRPC|
|VLESS-H2C-Caddy2|
|VLESS-TCP-TLS (maximal by rprx)|
|VLESS-TCP-TLS (minimal by rprx)|
|VLESS-TCP-TLS-WS (recommended)|
|VLESS-TCP-TLS-proxy protocol|
|VLESS-TCP-TLS|
|VLESS-TCP-XTLS-WHATEVER|
|VLESS-TCP|
|VLESS-mKCPSeed|
|VMess-HTTP|
|VMess-HTTP2|
|VMess-TCP-TLS|
|VMess-TCP|
|VMess-Websocket-TLS|
|VMess-Websocket|
|VMess-mKCPSeed|

**Comment by shenlijun** [comment]
主要是看你的vps快不快和客户机/软路由的cpu强不强

其它都是浮云

**Comment by shenlijun** [comment]
非繁忙时间我家里打开fast.com测速能1.2G，无论哪种方式

**Comment by maryjeck** [comment]
我们还是来讨论哪种方案最稳吧，快不快？反正不打游戏，无所谓了
